codespell_lib/data/dictionary.txt

```diff
@@ -52956,17 +53045,28 @@ treting->treating
 trew->true, threw,
 trewthful->truthful
 trewthfully->truthfully
+trgger->trigger
```

Comment:
Perhaps `tagger`, although probably uncommon?

Reply:
I couldn't find this typo.